### DISCUSSION TITLE
*SOLVED* network oddity with thin client only

### DISCUSSION THREAD

**Discussion by brcisna** [ltsp/ltsp#559] [General]
Hello All,

*SOLVED*
Placed an Intel network card in this new server box and this cured the WWAN , modem from disconnecting.
This is the Realtek RTL8111/###/#### giving the unstable connection, builtin on motherboard . 
As a sidenote for someone else maybe trying to troubleshoot this. I did try the r8168-dkms replacement module for this nic versus the in kernel r8169 module. . But it made the machine/network miserably slow.
Also what was obvious after seeing , on the old server still plugged into same network the new server nic was causing old server to hit 96% memory usage and is a 24 core poweredge with 126 GB ram, sitting idle !. never seen this before. !

Posted a similar question,but found a couple clues possibly.
New install of LTSP two nics. After about 15 to 30 minutes of internet use with a thin client the WWAN modem on server starts disconnecting reconnecting. This never happens with linux or windows hard drive machines connected to switch
As soon as i shut down client the modem straightens out, and the server has internet access again,,,instantly Have spent quite a bit of time trying a few different dnsmasq settings. What i did find in the ltsp debug log was the following. Could it be causing the problem?

21:13:44.897695455 Could not parse output of: ip -o route get 192.168.67.1
21:13:44.898718669 LTSP command failed: ip -o route get 192.168.67.2
21:13:44.899201655 Could not parse output of: ip -o route get 192.168.67.2
21:13:44.909164086 Disabled flow control for enp2s0 using mii-tool
21:13:44.910183947 Couldn't disable flow control for wlp3s0
21:13:44.911210486 Couldn't disable flow control for wwan0

Am using an ip range of 172.31.100.0/24
Is just have used this for umpteen years.
But what about not getting flow control disabled on wifi card,,,but more importantly the wwan?
Probably not at all possible on this Siemens modem?

Thank You.

**Comment by alkisg** [comment]
`ltsp service` disables ethernet flow control in order to make things faster on mixed-speed networks (e.g. gigabit server and one or more 100 mbps clients).

The warnings that you see in the logs are expected, there's no ethernet flow control on wifi networks, so it's OK that LTSP wasn't able to disable it (i.e. it did nothing at all).

You may disable the ltsp service with `sudo systemctl disable ltsp` and reboot to see if that helps.
